make/protogen.mk

```
@@ -37,15 +37,16 @@ endif
 PROTOC_VERSION := 21.4
 UNAME_S := $(shell uname -s)
 ifeq ($(UNAME_S),Linux)
-PROTOC_ARCH = linux
+PROTOC_OS = linux
 endif
 ifeq ($(UNAME_S),Darwin)
-PROTOC_ARCH = osx
+PROTOC_OS = osx
 endif
+PROTOC_ARCH=$(shell case $$(uname -m) in (aarch64) echo -n armv6 ;; (*) echo -n $$(uname -m) ;; esac)
```

Would just `uname -m` work for the default case?

```suggestion
PROTOC_ARCH=$(shell case $$(uname -m) in (aarch64) echo -n armv6 ;; (*) uname -m ;; esac)
```

---

Yeah, I don't think the `-n` for `echo` is providing any value. Also changed in the operator Makefile.